README.md

```diff
@@ -220,14 +220,57 @@ jasmine_node_test(
     name = "test",
     ...
     deps = [
-        "@npm//:jasmine",
-        "@npm//:foo",
-        "@npm//:baz",
+        "@npm//jasmine",
+        "@npm//foo",
+        "@npm//baz",
         ...
     ],
 )
 ```
 
+#### Fine-grained npm package nodejs_binary targets
+
+If an npm package lists one of more `bin` entry points in its `package.json`,
+`nodejs_binary` targets will be generated for these.
+
+For example, the `protractor` package has two bin entries in its `package.json`:
+
+```json
+  "bin": {
+    "protractor": "bin/protractor",
+    "webdriver-manager": "bin/webdriver-manager"
+  },
+```
+
+These will result in two generated `nodejs_binary` targets in the `@npm//protractor/bin`
+package (if your npm deps workspace is `@npm`):
+
+```python
+nodejs_binary(
+    name = "protractor",
+    entry_point = "protractor/bin/protractor",
+    data = ["//protractor:protractor"],
+)
+
+nodejs_binary(
+    name = "webdriver-manager",
+    entry_point = "protractor/bin/webdriver-manager",
+    data = ["//protractor:protractor"],
+)
+```
+
+These targets can be used as executables for actions in custom rules or can
+be run by Bazel directly. For example, you can run protractor with the
+following:
+
+```sh
+$ bazel run @npm//protractor/bin:protractor
+```
+
+Note: These targets are in the `protractor/bin` package so they don't
+conflict with the deps targets. For example, `@npm//protractor:protractor` is
+the deps targets while `@npm//protractor/bin:protractor` is the binary target.
```

Comment:
the deps targets -> the target to use in `deps[]`

Reply:
👍